platforms/dbaas/docs/installation.md

```diff
@@ -0,0 +1,4 @@
+
+# Installing and Configuring Splice Machine for dbaas profile
+
+The profile is for the cloud build service. Please refer to the cloud build confluence documentation in order to build and deploy this profile.
```

Comment:
And where is the _cloud build confluence documentation"_?! Why is there no link here? What's _"cloud build service"_?